### PR TITLE
[Android] Fix deprecated NotificationCompat.Builder(Context context)

### PR DESCRIPTION
**Pull request by adamradocz**
**Description of the Change**
Fix deprecated `Builder(android.content.Context)` and `android.support.v7.app.NotificationCompat.Builder`

The NotificationCompat.Builder(Context context) constructor was [deprecated](https://developer.android.com/reference/android/support/v4/app/NotificationCompat.Builder) in API level 26.1.0.
Google suggests to use NotificationCompat.Builder(Context, String) instead.

**Release Notes**
N/A

### CODE REVIEW

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java [hunk 1]
```diff
@@ -129,14 +129,14 @@ private Notification buildNotification() {
...
 
-        nb = (android.support.v7.app.NotificationCompat.Builder)(new NotificationCompat.Builder(this.context).
-            setContentTitle(this.context.getResources().getQuantityString(
-                R.plurals.notice_notification,
-                notices = this.currentlyNotifiedNotices.size(),
-                projectName = this.currentlyNotifiedNotices.get(0).project_name,
-                notices)).
-            setSmallIcon(R.drawable.mailw).
-            setContentIntent(this.contentIntent));
+		nb = new NotificationCompat.Builder(this.context, "main-channel");
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java]: @adamradocz, please fix indents

Comment (adamradocz) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java]: Resolved. I can't believe it always happens. :)

**Comment by AenBleidd** [comment]
Looks fine, Thanks. Merging...

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java (hunk 1 above)
Comment (RichardHaselgrove) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java]: Default setting in your editor? I use Notepad++, and after being prompted by similar requests, I found it in Settings --> Preferences --> Language --> Tab settings: Replace by space

Comment (adamradocz) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java]: What if we use the tab as standard?

Comment (RichardHaselgrove) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java]: I think you'd get a howl of protest. There is actually a stated common standard, which helps people navigate around in a comfortable environment:

https://boinc.berkeley.edu/trac/wiki/CodingStyle#indent

Comment (adamradocz) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/client/NoticeNotification.java]: Thanks

**Pull request merged**